backend/src/main/java/gov/cdc/usds/simplereport/service/PersonService.java

```diff
@@ -190,7 +190,7 @@ public List<Person> getPatients(
       int pageSize,
       boolean isArchived,
       String namePrefixMatch,
-      boolean includeArchivedFacilities) {
+      Boolean includeArchivedFacilities) {
```

Comment:
new Graphql lib likes objects, will not match on primitives, unless you supply a default value